src/etc/inc/plugins.inc.d/openvpn/wizard.inc

```diff
@@ -440,6 +440,12 @@ function step10_submitphpaction()
 
     if ($result = openvpn_validate_cidr($_POST['tunnelnet'], gettext('IPv4 Tunnel Network'), false, 'ipv4')) {
         $input_errors[] = $result;
+    } elseif (isset($_POST['tunnelnet'])) {
```

Comment:
`!empty($_POST['tunnelnet']) && strpos($_POST['tunnelnet'], "/") !== false` might be better.

Reply:
i thought strpos($_POST['tunnelnet'], "/") is already checked by openvpn_validate_cidr?

Reply:
you're right, didn't read into the function. But if tunnelnet is always there, a simple `else` would have been enough as well then I guess.